src/connectors/stats/__tests__/connectStats-test.js

```diff
@@ -119,6 +119,55 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/stats/js/#c
 
       stats.init(createInitOptions({ helper, state: helper.state }));
 
+      const results = new SearchResults(helper.state, [
+        createSingleSearchResponse(),
+      ]);
+
+      const renderState2 = stats.getWidgetRenderState(
+        createRenderOptions({
+          helper,
+          state: helper.state,
+          results,
+        })
+      );
+
+      expect(renderState2).toEqual({
+        hitsPerPage: results.hitsPerPage,
+        nbHits: results.nbHits,
+        nbPages: results.nbPages,
+        page: results.page,
+        processingTimeMS: results.processingTimeMS,
+        query: results.query,
+        widgetParams: {},
+      });
+    });
+
+    test('returns the widget render state with results', () => {
+      const renderFn = jest.fn();
+      const unmountFn = jest.fn();
+      const createStats = connectStats(renderFn, unmountFn);
+      const stats = createStats();
+      const helper = jsHelper(createSearchClient(), 'indexName', {
+        index: 'indexName',
+      });
+      helper.search = jest.fn();
+
+      const renderState1 = stats.getWidgetRenderState(
```

Comment:
these variables don't need 1 or 2 added to them anymore now 👍

Reply:
@Haroenv Let's hope this time this is what you meant 😂 
https://github.com/algolia/instantsearch.js/pull/4565/commits/5e07d5746c5bd60eeae6c4ce12fc8e975831e42e